scalableminds/webknossos
# in project statistics, correctly aggregate openInstances

In the project public writes, the total instances were wrongly shown, instead of the open instances.

### Steps to test:
- create a project and a task with  multiple instances
- take a task
- in Administration→Projects list, the Open Assignments collumn should now be decremented by one.

### Issues:
- fixes #2274

------
